config/routes.rb

```diff
@@ -0,0 +1,20 @@
+Rails.application.routes.draw do
+  root "splash#index"
+  
+  resources :splash, only: %i[index]
+  resources :categories, only: %i[index new create] do
+    resources :expenses, only: %i[index]
+  end
+
+  resources :expenses, only: %i[new create]
+
+  devise_for :users, controllers: { registrations: 'registrations' }
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
+  # Can be used by load balancers and uptime monitors to verify that the app is live.
+  get "up" => "rails/health#show", as: :rails_health_check
```

Comment:
Great job! The UI design looks fantastic. 👍

[OPTIONAL] - It would be great to include a feature that allows users to log out of the application.

- Logging out helps prevent other users from accessing the system without verifying their credentials. It also helps protect the current user's access or prevent unauthorized actions on the current login session and is thus an important part of security.

Reply:
Thank you @abdulhamiid 💯